docs/source/how-to/simulation_performance.rst

```diff
@@ -46,5 +46,11 @@ if the geometry is not part of a dynamic rigid body.
 Additional Performance Guides
 -----------------------------
 
+There are many ways to "tune" the performance of the simulation, but the way you choose largely defends on what you are trying to simulate. In general, the first place
+you will want to look for performance gains is with the `physics engine <https://docs.omniverse.nvidia.com/kit/docs/omni_physics/107.3/dev_guide/guides.html>`_. This is because
+specializing the engine to task will reduce the cost of something that must be called every newly rendered frame, so specializing the engine usually leads to large performance gains (in frame rate).
```

Comment:
Complex weird sentence phrasing. Can you please simplify this? What does updating physics engine have to be rendered frame? Do you just want to say stepping the simulator in general?